corehq/motech/repeaters/models.py

```diff
@@ -210,10 +216,13 @@ class SQLRepeater(models.Model):
         on_delete=models.PROTECT
     )
 
+    couch_id = models.CharField(max_length=126, null=True)
```

Comment:
I think it should be `unique`. Thanks for the suggestion @millerdev.

> There have been debates in the past about what to name fields like this one. SQLLocation calls it "location_id". CommCareCaseSQL calls it "case_id". Naming this "couch_id" suggests to me that we can drop this field once we have migrated all Repeaters off Couch. If that is not true -- in other words, if we are going to continue using this as a foreign key independent of Couch -- then I suggest we call it "repeater_id".

If we are going to use the default Django foreign keys to access the repeaters/repeat records then we won't be needing `repeater_id` unless we have a specific use case that depends on this id. I am not super familiar with its usage to call for the final verdict.

> My second nit is a little petty, so feel free to ignore this, because we ignore this a lot in the codebase. I think that max_length can be a useful indicator of what is stored in the field. These are UUIDs, so they are never going to be longer than 36 characters. If the value is longer than 36 characters, that would be something we'd want to know about, and so it should raise a validation error.

I would do +1 to that, I just looked at other places where it was set to 126, so I kept it that way.